Package@swift-4.swift

```diff
@@ -0,0 +1,18 @@
+// swift-tools-version:4.0
+
+import PackageDescription
+
+let package = Package(
+    name: "CSRF",
+    products: [
+        .library(name: "CSRF", targets: ["CSRF"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/vapor/vapor.git", from: "2.0.0"),
+        .package(url: "https://github.com/nodes-vapor/flash", from: "1.0.0"),
```

Comment:
I don't think this dependency is needed?

Reply:
I was unable to find where it was used in the code. Should I remove it?

Reply:
Sounds good to me! And from the original Package.swift as well

Reply:
Done.